test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -25,15 +25,15 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: matched
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
 
         //partial word: not matched
         assertFindCommandBehavior(new String[]{"my"}, Collections.emptyList());
 
         //multiple words: matched
         assertFindCommandBehavior(new String[]{"Amy", "Bill", "Candy", "Destiny"},
-                Arrays.asList(td.amy, td.bill, td.candy));
+                Arrays.asList(td.amy, td.bill, td.candy));//N
```

Comment:
Unnecessary comment?

Reply:
Hi,
 I was just trying to add the Junit, following what my friend does. Apparently it did not work out.

Reply:
That is fine. To avoid such errors in the future, it is recommended to check the changes before submitting the Pull Request. Good Job anyway! Please read the comments before closing this PR.